presto-postgresql/pom.xml

```diff
@@ -85,6 +85,14 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
+            <!-- TODO remove this after fixing https://github.com/prestosql/presto/issues/5718 -->
```

Comment:
I think we should do that issue instead.,

Reply:
I would prefer not. Guava could be a pandora box.